### DISCUSSION TITLE
Delete a suspend account

### DISCUSSION THREAD

**Discussion by ltdora** [community/community#136539] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

Hello, i had a suspend account. I want to delete my suspend account to create a new account use this account.
Anyone can help me? I don't know what i have to do.
Thanks so much.

**Comment by danialtavakoli** [comment]
If your account is suspended, you won't be able to directly delete it or create a new account using the same email address. The best course of action is to contact the support team of the platform where your account is suspended. Explain your situation and request assistance with either reactivating your suspended account or deleting it so you can create a new one. Usually, the support team will guide you through the steps or provide specific instructions based on their policies.
